weather.py

```diff
@@ -79,8 +106,65 @@ def find_max(weather_data):
     Returns:
         The maximum value and it's position in the list.
     """
-    pass
-
+    # if len(weather_data) == 0:
+    #     return()
+    # index = 1
+    # result = [float(weather_data[0]), 0]
+    # for item in weather_data[1:]:
+    #     if float(item) >= result[0]:
+    #         result = [float(item), index]
+    #     index += 1
+    # return(result[0], result[1])
+    if len(weather_data) == 0:
+        return()
+    index = 1
+    max = float(weather_data[0])
+    max_index = 0
+    for item in weather_data[1:]:
+        if float(item) >= max:
+            max = float(item)
+            max_index = index
+        index += 1
+    return(max, max_index)
+
+
+
+
+def format_min_summary(weather_data):
+    def get_min(weather_line_item):
+        return weather_line_item[1]
+    min_val = list(map(get_min, weather_data))
+    min_val_index = find_min(min_val)[1]
+    min_weather_line_item = weather_data[min_val_index]
+    min_c = format_temperature(convert_f_to_c(min_weather_line_item[1]))
```

Comment:
Nice work applying `map()`.
Since `find_min` returns a tuple output, we can also use tuple unpacking so we don't need to work with indexing. e.g.
```suggestion
min_temp, min_val_index = find_mind(min_val)
min_c = format_temperature(convert_f_to_c(min_temp))
```